### DISCUSSION TITLE
Initialization and run problems

### DISCUSSION THREAD

**Discussion by torodecamargue** [nextfoam/baram#75] [General]
Hello,

I have done several times the same problem, mesh BC etc.. But the solver failed. First times it was due to a bad initialization.
But no message in the console

![image](https://github.com/nextfoam/baram/assets/75060318/c4b0d463-3a90-4067-a24b-0857c3a2c471)

So when I try to initialize the case it never ends. I don't know why.

![image](https://github.com/nextfoam/baram/assets/75060318/d36d03c4-4b93-489c-84da-82dd6f7695bb)

Do you know what could be wrong? 

I see also this error during initialization

![image](https://github.com/nextfoam/baram/assets/75060318/f4dfd01c-a1e3-4d02-bb36-3811514b113f)

Best regards

Julien

**Comment by torodecamargue** [comment]
It seems to be linked with parallel run settings.
With mono processor it starts.

Is there a procedure to know when decompose the case (settings to x processors), just before run the case or at the beginning? I tryed different way to do, sometimes it had work. It seems quite random, Sometimes when setting the number of processors I need to close Baram, decompose never ends.

Best regards

Julien

**Comment by kdg5424** [comment]
Hello Julien. This is donggyu.
The problem doesn't happen in my PC. So we need your case and log file.
You can see log, stdout.log or stderr.log on the case folder.

![image](https://github.com/nextfoam/baram/assets/133930861/2a9c41c3-eacf-4b19-ba66-1ef91e383a31)

![image](https://github.com/nextfoam/baram/assets/133930861/39398e50-fd44-470c-b3f1-5af50a031fe5)

If you're ok, please send us your case and log file.
If it's not possible to send the case file, then just send us your log file please.
Then, jake and I will check it.

Regards

Donggyu

**Comment by torodecamargue** [comment]
Hello Donggyu,

See my log files, the stderr and stdout files were not generated.
I attached 2 log files corresponding at a bad decomposition (for parallel runs) and a bad initialization

I had to crash Baram when I enter in a command that never ends running initialization or decomposition.

[log.txt](https://github.com/nextfoam/baram/files/15429415/log.txt)

[log.txt](https://github.com/nextfoam/baram/files/15429424/log.txt)

Best regards

Julien

**Comment by torodecamargue** [comment]
I find a new problem.

I decompose the case in BaramFlow, After that I run the case and I have the error message

![image](https://github.com/nextfoam/baram/assets/75060318/7197cc9c-ee61-4cee-baf9-7db03b5133c6)

[log.txt](https://github.com/nextfoam/baram/files/15437486/log.txt)
[stderr.log](https://github.com/nextfoam/baram/files/15437489/stderr.log)
[stdout.log](https://github.com/nextfoam/baram/files/15437490/stdout.log)

Best regards

**Reply by MTODC** [reply]
Hello,
Is there any update about this issue? I have a quite similar problem, but it happens even if it runs in single-core.

**Reply by kdg5424** [reply]
Hello @MTODC .

There are some problems with B.C. and mesh topology between the porous jump.
We've updated it in BARAM v24.3

Thank you

**Comment by kdg5424** [comment]
Hello Julien.
I'm sorry for your inconvenience.
If you're ok, could you send us your case file?

After compressing your case file, upload on this post.
Then we'll check it.

If the file size is so large to attach here, you could send us email.
This is my email.
dkkim@nextfoam.co.kr

Thanks

**Comment by torodecamargue** [comment]
Hello,

Find the Mesh attached. Let me know if you need more information.

[Mesh_oven.zip](https://github.com/nextfoam/baram/files/15451975/Mesh_oven.zip)

Best regards

Julien

**Comment by torodecamargue** [comment]
Do you know how to resolve this error?

![image](https://github.com/nextfoam/baram/assets/75060318/291f8bff-0861-4942-b3d5-3f533a33d381)

Where can I increase this tolerance?

Best regards

Julien

**Comment by torodecamargue** [comment]
I found where I can modify manually this tolerance.

Thank you

**Comment by kdg5424** [comment]
Hi Julien.

Sorry for late reply. I'm on the business trip now.
So after business trip, let me check.
Sorry for late respond.

Regards Donggyu kim

**Comment by kdg5424** [comment]
Hello Julien.
I can't find the polyMesh with your case file. Could you check it?
Thanks.
